src/30_Advanced/SeatMap.html

```diff
@@ -60,17 +62,23 @@
       #pan-zoom {
         display: initial;
       }
-      #pan-zoom > svg {
-        left: 0;
-        right: 0;
-        margin: auto;
+      #pan-zoom >  amp-selector {
+          left: 0;
+          right: 0;
+          margin: auto;
+      }
+      #pan-zoom > amp-selector [option][selected] {
+          outline: none;
       }
       .list-container {
         position: relative;
         height: 50vh;
         min-width: 100%;
         box-sizing: border-box;
       }
+      .svg-placeholder {
```

Comment:
You don't need to create a new class for this, you can target via: 

`[placeholder] > svg`

Reply:
done